### PR TITLE
Reading UUID from URL Ending

### PR DESCRIPTION
**Pull request by SUmarFarooq**
Ticket [here](https://app.asana.com/0/1173952391708705/1198303882370429).

**TLDR:** The task ARN now seems to have the primary cluster before the name. Can't find anywhere this change is documented from AWS, so below is an explanation of my rationale/process.

We pull the UUID from the Task ARN. For a [recent log message](https://us-east-2.console.aws.amazon.com/cloudwatch/home?region=us-east-2#logsV2:log-groups/log-group/$252Faws$252Flambda$252Fprimary-cluster-ecs-events-to-slack/log-events/2020$252F10$252F21$252F$255B$2524LATEST$255D1264e285395b4332b0fb4de27d070efd), the ARN is: 

```
arn:aws:ecs:us-east-2:667471847564:task/primary-cluster/8af2f60e088540e6981351e8ceed7ecb
```

For an [older ARN](https://us-east-2.console.aws.amazon.com/cloudwatch/home?region=us-east-2#logsV2:log-groups/log-group/$252Faws$252Flambda$252Fprimary-cluster-ecs-events-to-slack/log-events/2019$252F11$252F25$252F$255B$2524LATEST$255D12e40af1e8af4342abba154b8f7ba63e), it looks like: 

```
arn:aws:ecs:us-east-2:667471847564:task/117c2ec5-d3ca-404b-850c-118155ca627f
```

Unfortunately I'm not able to find any recent logs from before 10/13/2020 in Cloudwatch for the lambda event (the log streams don't seem to have entries between 10/16/2020 and 11/25/2019) so my logs example is pretty old. Still, it looks like recent messages support this. An Apiserver-staging [message](https://alloy-us.slack.com/archives/CNXQCK2RY/p1602622190315600) from 10/13 have log URLs ending in `/apiserver-staging/a6c4dc78-4184-43ee-aa71-e7fe5acdb21f` but more [recent messages](https://alloy-us.slack.com/archives/CNXQCK2RY/p1603290630095600) have log URLs that end with `/apiserver-staging/primary-cluster`.

### CODE REVIEW

File: lambdas/ecs-events-to-slack/notify.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def message_formatter(event, region):
 
     event_detail = event["detail"]
     task_arn = event_detail["taskArn"]
-    task_id = task_arn.split("/")[1]
+    task_id = task_arn.split("/")[2]
```
Comment (alex) [inline lambdas/ecs-events-to-slack/notify.py:27]: Can you add a comment to this line with the format that `task_arn` is in, so the next person has a shot at debugging :-)

**Review by alex — APPROVED** [review]
Perfect. I can't quite remember how we deploy this, so you might need to sync with @scout

**Comment by SUmarFarooq** [comment]
I do not have permissions to merge this PR in, so if you approve it, please also merge it in. Thank you!

**Review by alex — APPROVED** [review]
